docs/docs/integrations/providers/cohere.mdx

```diff
@@ -4,62 +4,74 @@
 > that help companies improve human-machine interactions.
 
 ## Installation and Setup
-- Install the Python SDK :
+- Install the Cohere Python SDK and the LangChain partner package :
 ```bash
-pip install cohere
+pip install cohere langchain-cohere
 ```
 
 Get a [Cohere api key](https://dashboard.cohere.ai/) and set it as an environment variable (`COHERE_API_KEY`)
 
 ## Cohere langchain integrations
 
-|API|description|Endpoint docs|Import|Example usage|
-|---|---|---|---|---|
-|Chat|Build chat bots|[chat](https://docs.cohere.com/reference/chat)|`from langchain_community.chat_models import ChatCohere`|[cohere.ipynb](/docs/integrations/chat/cohere)|
-|LLM|Generate text|[generate](https://docs.cohere.com/reference/generate)|`from langchain_community.llms import Cohere`|[cohere.ipynb](/docs/integrations/llms/cohere)|
-|RAG Retriever|Connect to external data sources|[chat + rag](https://docs.cohere.com/reference/chat)|`from langchain.retrievers import CohereRagRetriever`|[cohere.ipynb](/docs/integrations/retrievers/cohere)|
-|Text Embedding|Embed strings to vectors|[embed](https://docs.cohere.com/reference/embed)|`from langchain_community.embeddings import CohereEmbeddings`|[cohere.ipynb](/docs/integrations/text_embedding/cohere)|
-|Rerank Retriever|Rank strings based on relevance|[rerank](https://docs.cohere.com/reference/rerank)|`from langchain.retrievers.document_compressors import CohereRerank`|[cohere.ipynb](/docs/integrations/retrievers/cohere-reranker)|
+|API|description|Endpoint docs|
+|---|---|---|
+|Chat|Build chat bots|[chat](https://docs.cohere.com/reference/chat)|
```

Comment:
Why drop the example imports?

Reply:
I did not think it added any extra information - the code examples below for each component have it.

if you feel that it is helpful, I'll add it back. (in general I agree that more information in the documentation is better even if it is repeated)